packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -38,9 +38,15 @@ const selectors = {
 
 	// Publish panel (including post-publish)
 	publishPanel: '.editor-post-publish-panel',
+	// With the selector below, we're targeting both "View Post" buttons: the one
+	// in the post-publish pane, and the one that pops up in the bottom-left
+	// corner. This addresses the bug where the post-publish panel is immediately
+	// closed when publishing with certain blocks on the editor canvas.
+	// See https://github.com/Automattic/wp-calypso/issues/54421.
 	viewButton: 'text=/View (Post|Page)/',
 	addNewButton: '.editor-post-publish-panel a:text-matches("Add a New P(ost|age)")',
 	closePublishPanel: 'button[aria-label="Close panel"]',
+	snackbarViewButton: 'a.components-snackbar__action:has-text("View Post"):visible',
```

Comment:
What is this selector for? I didn't see it being used anywhere

Reply:
Ah, must have been left over. Thanks!

Reply:
Removed.